packages/package_bumbershoot_3_0_8738/.shed.yml

```diff
@@ -0,0 +1,10 @@
+categories:
+- Tool Dependency Packages
+description: Contains a tool dependency definition that downloads the Bumbershoot tools.
+long_description: |
+  Bumbershoot
+
+  https://sourceforge.net/p/proteowizard/code/HEAD/tree/trunk/pwiz/pwiz_tools/Bumbershoot/
+name: package_bumbershoot_3_0_8738
```

Comment:
remote_repository_url: https://github.com/galaxyproteomics/tools-galaxyp/packages/package_bumbershoot_3_0_8738/

is missing.